CHANGELOG.md

```diff
@@ -5,34 +5,39 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#63)
 
 New features:
+- Replaced monomorphic proxies with `Type.Proxy.Proxy` and polymorphic variables and made `If` and `TypeEquals` polykinded (#64)
 
 Bugfixes:
+- Re-add re-exports of `Ordering` and `Boolean` (#55)
 
 Other improvements:
+- Migrated CI to GitHub Actions and updated installation instructions to use Spago (#67)
+- Added a changelog and pull request template (#68, #69)
 
```

Comment:
My script didn't include the `v5.0.2` release, which only had [one change from #56](https://github.com/purescript/purescript-typelevel-prelude/compare/v5.0.1...v5.0.2). The following markdown should work. You can copy and paste it here.

```
## [v5.0.2](https://github.com/purescript/purescript-typelevel-prelude/releases/tag/v5.0.2) - 2020-01-18

- Re-adds the re-exports of the Ordering and Boolean kinds to the module `Type.Prelude` (#56)
```

Reply:
This would also entail removing the bugfix above where the #55 is included.